db/config.hh

```diff
@@ -349,6 +349,7 @@ public:
     named_value<size_t> large_memory_allocation_warning_threshold;
     named_value<bool> enable_deprecated_partitioners;
     named_value<bool> enable_keyspace_column_family_metrics;
+    named_value<bool> enable_node_table_metrics;
```

Comment:
We can drop the word "node", it doesn't add anything.

Reply:
We have per-shard table we can enable and disable.
This configuration, came from a request by @tzach to disable this metric group, node here is to mark that those are per-node-per-table

Reply:
It's not an intuitive name.

Reply:
@avikivity how about ```node-level-table-metrics``` ?

Reply:
node-aggregated-table-metrics, indicates the node's role in this